src/components/pages/deployments/deploymentDetails/deploymentDetails.js

```diff
@@ -114,18 +106,22 @@ export class DeploymentDetails extends Component {
     } = this.props;
     const {
       id,
-      appliedCount,
-      targetedCount,
-      succeededCount,
-      failedCount,
+      appliedCount = Config.emptyValue,
```

Comment:
It feels wrong to put a string into a variable which is clearly a number.
Could you utilize a cell renderer or other utility function (renderUndefined) to do this at display time instead?